doc/source/user_guide/hpc/launch_mapdl_entrypoint.rst

```diff
@@ -0,0 +1,235 @@
+
+.. _ref_pymapdl_interactive_in_cluster_hpc:
+
+.. _ref_pymapdl_interactive_in_cluster_hpc_from_login:
+
+
+Interactive MAPDL instance launched from the login node
+=======================================================
+
+Starting the instance
+---------------------
+
+If you are already logged in an login node, it is also
+possible to launch an MAPDL instance as an SLURM job and
+connect to it.
+This can be accomplished running the following commands
+in your login node.
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> mapdl = launch_mapdl(launch_on_hpc=True)
+
+PyMAPDL submits a job to the scheduler using the appropriate commands.
+In case of SLURM, it uses ``sbatch`` command with the ``--wrap`` argument
+to pass the MAPDL command line need to start.
+Other scheduler arguments can be specified using ``scheduler_options``
+argument as a Python :class:`dict`:
```

Comment:
```suggestion
In case of SLURM, it uses the ``sbatch`` command with the ``--wrap`` argument
to pass the MAPDL command line to start.
Other scheduler arguments can be specified using the ``scheduler_options``
argument as a Python :class:`dict`:
```